### PR TITLE
TM-6365 add long desc and code to bureau filter

### PR DESCRIPTION
**Pull request by fimbriano**
😺
Dual Merge: 
- [MOCK PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/502)

[Ticket](https://metaphase.atlassian.net/browse/TM-6365)

Updated API for standardization.

WS updated GET /api/v1/backoffice/BackOfficeCRUD to return new info

### CODE REVIEW

File: talentmap_api/fsbid/services/publishable_positions.py [hunk 1]
```diff
@@ -271,7 +271,9 @@ def cycle_map(x):
         }
     def bureau_map(x):
         return {
-            'description': x.get('ORGS_SHORT_DESC'),
+            'code': x.get('BUREAU_CD'),
+            'short_description': x.get('ORGS_SHORT_DESC'),
+            'description': x.get('ORGS_LONG_DESC'),
```
Comment (M00staff) [inline talentmap_api/fsbid/services/publishable_positions.py:276]:
```suggestion
            'short_description': x.get('BUREAU_SHORT_DESC'),
            'description': x.get('BUREAU_LONG_DESC'),
```

Comment (M00staff) [inline talentmap_api/fsbid/services/publishable_positions.py:276]: i don't think this is right - nor, should it have worked. 

this line `'bureauFilters': list(map(bureau_map, data.get('QRY_LSTBUREAUS_DD_REF'))),` 

is grabbing values from this part of the mock data 

image

which doesn't return `ORGS_LONG_DESC` or `ORGS_SHORT_DESC` 

so, it should have just been returning undefined or null here ... also, ORGs and Bureaus are different (even though they are labeled confusingly... 

you'll have to double check swagger and possibly ask WS to update this endpoint as the PPos filters are not returning the Bureau long and short descriptions, the same way that the Projected Vacancy filters are

Comment (M00staff) [reply talentmap_api/fsbid/services/publishable_positions.py:276]: derp - just read the description 🤦

make sure to update the mock (and link it to your PR) also make sure to double check that WS made the correct changes and the field names are correct 

still not sure why those fields are ORGs 

the PV filters look like this
image

**Review by M00staff — CHANGES_REQUESTED** [review]
update the mock - make sure you're getting Bureau data and not Org data

File: talentmap_api/fsbid/services/publishable_positions.py (hunk 1 above)
Comment (M00staff) [reply talentmap_api/fsbid/services/publishable_positions.py:276]: OH - and you prolly need to ask someone for the 'sprint sheet' that has the stored procedure endpoints on it - and how to hit those endpoints in swagger

Comment (fimbriano) [reply talentmap_api/fsbid/services/publishable_positions.py:276]: I had asked web services to grab the same info bureau permissions grabs for the bureau code. 

I did forget to update the mock data tho haha i will do that

**Review by M00staff — APPROVED** [review]
just add a comment explaining the Bureau/Location(Org) thing we chatted about and GtG

**Pull request merged**